qt/scientific_interfaces/ISISReflectometry/test/Experiment/ExperimentOptionDefaultsTest.h

```diff
@@ -141,6 +141,16 @@ class ExperimentOptionDefaultsTest : public CxxTest::TestSuite {
 
   void testInvalidCorrectionOptionsFromParamsFile() { getDefaultsFromParamsFileThrows("Correction_Invalid"); }
 
+  void testDefaultStitchParamsOptions() {
+    auto result = getDefaults();
+    TS_ASSERT_EQUALS(result.stitchParameters().empty(), true);
+  }
+
+  void testValidStitchParamsOptionsFromParamsFile() {
+    auto result = getDefaultsFromParamsFile("Experiment");
+    TS_ASSERT_EQUALS(result.stitchParameters().size(), 1);
```

Comment:
Could we add a check that the right name and value have been taken from the IPF?